src/pe_source/data/sixgill/source.py

```diff
@@ -97,9 +101,33 @@ def alerts(org_id):
         all_alerts.append(df_alerts)
         df_all_alerts = pd.concat(all_alerts).reset_index(drop=True)
 
+    # Fetch the full content of each alert
+    # for i, r in df_all_alerts.iterrows():
+    #     print(r["id"])
+    #     content = alerts_content(org_id, r["id"])
+    #     df_all_alerts.at[i, "content"] = content
+
```

Comment:
Can these lines be deleted?

Reply:
resolved: [036b1d9](https://github.com/cisagov/pe-reports/pull/193/commits/036b1d9ca88dc6966e5914e5ae658140e7ec568c)